src/app/widgets/inlineinput/inlineinput.component.ts

```diff
@@ -77,11 +77,13 @@ export class InlineInputComponent implements OnInit {
   }
 
   closeClick() {
```

Comment:
This method should validate the input value

Reply:
Why should it? I am closing the field here, not saving.